src/core/server/saved_objects/types.ts

```diff
@@ -112,6 +113,14 @@ export interface SavedObjectsFindOptions {
   preference?: string;
   /** If specified, will find all objects belong to specified workspaces **/
   workspaces?: string[];
+  /**
+   * The params here will be combined with bool clause and is used for filtering with ACL structure.
+   */
+  ACLSearchParams?: {
+    workspaces?: string[];
```

Comment:
Cannot really remember the difference of `ACLSearchParams.workspaces` vs `SavedObjectsFindOptions.workspaces`, could you help me to recall? :D

Reply:
From the code, the `SavedObjectsFindOptions.workspaces` support pass `*` to match workspaces. The `ACLSearchParams.workspaces` doesn't support that. @SuZhou-Joe Could we remove the `ACLSearchParams.workspaces`? It's a little bit confused.